### ISSUE TITLE
Does the Lion optimizer work with grad accumulation?

### ISSUE THREAD

**Issue by IwanVan** [lucidrains/lion-pytorch#25]
Hey mates,

thanks for your great work. I am wondering now if the Lion optimizer will work with grad accumulation in pytorch (using transformer API for NLP tasks)? 

I read some work, which is built on Lion, but enabled the gradient accumulation, by setting `β1=β2=β` as a special case of Lion. The author named the optimizer as “Tiger” (Tight-fisted Optimizer), here is the official [implementation](https://github.com/bojone/tiger) (you might need to translate the website into English with Chrome) and the rewrite in [pytorch](https://github.com/hazdzz/Tiger).

**Comment by xiangning-chen** [comment]
Hi, thanks for the reference, yeah I think that Lion works with gradient accumulation, beta1=beta2 is a special case.
